code/modules/jobs/job_types/engineering.dm

```diff
@@ -145,6 +146,7 @@ Atmospheric Technician
 	box = /obj/item/weapon/storage/box/engineer
 	pda_slot = slot_l_store
 	backpack_contents = list(/obj/item/device/modular_computer/tablet/preset/advanced=1)
+	l_hand = /obj/item/weapon/tank/internals/rebreather

```

Comment:
1. When I wrote this I assumed that they don't fit in backpacks
2. It's a very useful item, so hopefully people will try to use it if it's in their hand